rasa/train.py

```diff
@@ -84,7 +84,10 @@ def train(
     try:
         domain = Domain.load(domain, skill_imports)
     except InvalidDomain as e:
-        print_error(e)
+        print_error(
+            "Could not load domain due to: '{}'. To specify a valid domain path use "
+            "the '--domain' argument.".format(e)
```

Comment:
this is just a thought and maybe doesn't go in this PR, but we should really in `InvalidDomain` check if it's a `YamlParserError` and if so lead them to YAMLint bc i've seen so many git issues from incorrectly formatted domain ymls is

Reply:
Do you want to open an issue for that? Maybe I have time to tackle it today or tomorrow.